.github/workflows/build_daily_dbt_models.yaml

```
@@ -0,0 +1,58 @@
+name: build-daily-dbt-models
+
+on:
+  schedule:
+    # Every day at 8am (2pm UTC-5)
+    - cron: '0 13 * * *'
+
+jobs:
+   build-daily-dbt-models:
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint
+    # so that we can authenticate with AWS
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup dbt
+        uses: ./.github/actions/setup_dbt
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
+          role-duration-seconds: 900
+
+      - name: Restore dbt state cache
+        id: cache
+        uses: ./.github/actions/restore_dbt_cache
+        with:
+          path: ${{ env.PROJECT_DIR }}/${{ env.STATE_DIR }}
+          key: ${{ env.CACHE_KEY }}
+
+      - name: Build daily models
+        run: dbt run -t "$TARGET" -s tag:daily --defer --state "$STATE_DIR"
+        working-directory: ${{ env.PROJECT_DIR }}
+        shell: bash
+
+      - name: Test daily models
+        run: dbt test -t "$TARGET" -s tag:daily --defer --state "$STATE_DIR"
+        working-directory: ${{ env.PROJECT_DIR }}
+        shell: bash
+
```

@jeancochrane We weren't sure if we needed to update the state cache here. I'm not sure that anything in the cache would change since the asset is already supposed to be built.

```suggestion

      - if: github.ref == 'refs/heads/master' && success()
        name: Update dbt state cache
        uses: ./.github/actions/save_dbt_cache
        with:
          path: ${{ env.PROJECT_DIR }}/${{ env.TARGET_DIR }}
          key: ${{ env.CACHE_KEY }}
          
```